server_addon/deadline/server/settings/publish_plugins.py

```diff
@@ -211,6 +211,49 @@ class HarmonySubmitDeadlineModel(BaseSettingsModel):
     department: str = SettingsField(title="Department")
 
 
+class HoudiniSubmitCacheDeadlineModel(BaseSettingsModel):
+    priority: int = SettingsField(title="Priority")
+    chunk_size: int = SettingsField(title="Chunk Size")
+    group: str = SettingsField(title="Group")
+
+
+class HoudiniSubmitExportDeadlineModel(BaseSettingsModel):
+    priority: int = SettingsField(title="Priority")
+    chunk_size: int = SettingsField(title="Chunk Size")
+    group: str = SettingsField(title="Group")
+
+
+class HoudiniSubmitRenderDeadlineModel(BaseSettingsModel):
+    priority: int = SettingsField(title="Priority")
+    chunk_size: int = SettingsField(title="Chunk Size")
+    group: str = SettingsField(title="Group")
+
+
+class HoudiniSubmitDeadlineModel(BaseSettingsModel):
+    """Houdini deadline submitter settings."""
+    enabled: bool = SettingsField(title="Enabled")
+    optional: bool = SettingsField(title="Optional")
+    active: bool = SettingsField(title="Active")
+    HoudiniSubmitCacheDeadline: HoudiniSubmitCacheDeadlineModel = (
```

Comment:
Why do you nest them under single model? You then have to do this https://github.com/ynput/ayon-core/pull/10/files#diff-e7f176c29a36e99c26e9934dd0a1acfab0b28dc7f5423f7edc85735f871456c0R88

Reply:
Oh, I was concerned about organising the Houdini settings and missed that!